<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="DialogButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>451</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить файл</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Extension">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>451</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Маска входного файла</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DeleteInput">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Удалять входной файл</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="OutputFilePath">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>451</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать директорию выходного файла</string>
    </property>
   </widget>
   <widget class="QLabel" name="OutputFilePathLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Путь</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="RewriteFile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Перезаписать файл</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="OutputExtension">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>451</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Имя выходного файла</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Timer">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>451</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Период опроса входного файла</string>
    </property>
   </widget>
   <widget class="QLabel" name="Result">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="FileCheck">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Таймер</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Variable">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Переменная для изменений 0...255</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Operation">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>OR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>XOR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AND</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
